keyserver/coniksserver/cmd/init.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"strconv"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+	"github.com/coniks-sys/coniks-go/crypto/vrf"
+	"github.com/coniks-sys/coniks-go/keyserver"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	"github.com/coniks-sys/coniks-go/utils"
+	"github.com/spf13/cobra"
+)
+
+// initCmd represents the init command
+var initCmd = &cobra.Command{
+	Use:   "init",
+	Short: "Create a configuration file and generate all keys",
+	Long:  `Create a configuration file and generate all keys for signing and VRF`,
+	Run: func(cmd *cobra.Command, args []string) {
+		dir := cmd.Flag("dir").Value.String()
+		mkConfig(dir)
+		mkSigningKey(dir)
+		mkVrfKey(dir)
+
+		cert, err := strconv.ParseBool(cmd.Flag("cert").Value.String())
+		if err == nil && cert {
+			testutil.CreateTLSCert(dir)
```

Comment:
If `CreateTLSCert` won't be used only in tests anymore we should move it from `testutil` into a general `util` package.